dandi/cli/cmd_validate.py

```diff
@@ -20,36 +22,46 @@
     is_flag=True,
     help="Whether to write a report under a unique path in the DANDI log directory.",
 )
+@click.option(
+    "--grouping",
+    "-g",
+    help="How to group error/warning reporting.",
+    type=click.Choice(["none", "path"], case_sensitive=False),
+    default="none",
+)
 @click.argument("paths", nargs=-1, type=click.Path(exists=True, dir_okay=True))
 @map_to_click_exceptions
 def validate_bids(
     paths,
     schema,
     report,
     report_path,
+    grouping="none",
 ):
     """Validate BIDS paths.
 
+    Parameters
+    ----------
+
+    grouping : str, optional
+        A string which is either "", "error", or "path" — "error" implemented.
+
```

Comment:
```suggestion
```

absolutely no point to have it here since we rely on `click` to provide all docs etc, why duplicate?

Reply:
done 677ef33e582c8b7225c29e567e206eeacb1335c3